lib.php

```diff
@@ -635,3 +635,17 @@ function equella_grade_item_delete($eq) {
     return grade_update(EQUELLA_SOURCE, $eq->courseid, EQUELLA_ITEM_TYPE, EQUELLA_ITEM_MODULE, $eq->id, 0, NULL, array('deleted'=>1));
 }
 
+/**
+ * Select the corresponding field to allocate the username to $USER
+ */
+function mod_equella_after_config(){
```

Comment:
I am not sure if you need this callback. The configured SSO identification field is available in `$CFG->equella_userfield`.

Reply:
Hi,
Because of what Dan explained here https://github.com/openequella/moodle-mod_openEQUELLA/issues/40 
The config data are not reloaded unless you purge the cache and logout/login again. This hook allows to update the config from $CFG.